components/speedreader/rust/ffi/Cargo.lock

```diff
@@ -1291,18 +1291,18 @@ checksum = "1ee22067b7ccd072eeb64454b9c6e1b33b61cd0d49e895fd48676a184580e0c3"
 
```

Comment:
the file shouldn't be in the PR

Reply:
yea, I'm not sure how it ended up in there